packages/contentstack-bulk-publish/src/producer/cross-publish.js

```diff
@@ -198,8 +207,19 @@ async function getSyncEntries(
         );
       }
 
+      if (variantsFlag) {
+        let entries = entriesResponse.items;
+        for (let index = 0; index < entries.length; index++) {
+          let variants = [];
+          variants = await getVariantEntries(stack, entries[index].content_type_uid, entries, index, queryParamsObj);
```

Comment:
@sunil-lakshman , pls avoid unnecessary variable declaration. 

        for (let index = 0; index < entriesResponse?.items?.length; index++) {
          const variants = await getVariantEntries(stack, entries[index].content_type_uid, entries, index, queryParamsObj);